packages/cli/lib/commonOpts.js

```diff
@@ -66,8 +69,16 @@ const setLogLevel = (options = {}) => {
   const { debug } = options;
   if (debug) {
     Logger.setLogLevel(LOG_LEVEL.DEBUG);
+
+    // Update the log level in local-dev-lib's instance of the logger
+    // This will evenutally replace cli-lib's version of it
+    setLocalDevLibLogLevel(LOG_LEVEL.DEBUG);
```

Comment:
I realized that certain log statements in local-dev-lib were not working correctly. It was because the default log level is `NONE`, and we were only setting the log level to `LOG` for the cli-lib instance of the logger.